modules/system/console/MixWatch.php

```diff
@@ -106,6 +112,11 @@ protected function createWebpackConfig(string $mixJsPath): void
      */
     public function getSubscribedSignals(): array
     {
+        //Return empty subscribed signals to the Symfony Console Application Bootloader.
+        if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
+            return [];
```

Comment:
The conflicting signal handling code merged with @bennothommo's commit https://github.com/wintercms/winter/commit/452bb4222351b1a6d3cc0288a30804ebeb37234d